### PR TITLE
return EMPTY instead of error

### PR DESCRIPTION
**Pull request by tltk90**

### CODE REVIEW

**Review by Danaorlov1005 — APPROVED** [review]

File: src/app/@ansyn/imagery/model/base-imagery-map.ts [hunk 1]
```diff
@@ -106,7 +106,7 @@ export abstract class BaseImageryMap<T = any> {
 	}
 
 	fitToExtent(extent: any): Observable<any> {
-		throw new Error('Method not implemented.');
```
Comment (ramtob) [inline src/app/@ansyn/imagery/model/base-imagery-map.ts:109]: Isn't it better to implement this function inside openlayers-disabled-map?

**Review by ramtob — COMMENTED** [review]
👍

File: src/app/@ansyn/imagery/model/base-imagery-map.ts (hunk 1 above)
Comment (tltk90) [reply src/app/@ansyn/imagery/model/base-imagery-map.ts:109]: we also have cesium-map(and maybe more in the future),
I think that is better to implement it here.
and map that want to change this behavior, will implement this function.

Comment (ramtob) [reply src/app/@ansyn/imagery/model/base-imagery-map.ts:109]: Ok

**Review by ramtob — APPROVED** [review]

**Pull request merged**